src/Services/Cvss.php

```diff
@@ -6,27 +6,26 @@
 
 class Cvss
 {
+    static private function betweenValue($value, $start, $end)
```

Comment:
This codebase is PHP7 and I would encourage you to use typehinting for the method arguments and the return value as well. This makes the code more robust and helps us catch errors earlier.
Thanks!
eg
```suggestion
    static private function betweenValue(float $value, float $start, float $end): bool
```